software_training/.gitignore

```diff
@@ -0,0 +1 @@
+
```

Comment:
this file should be used to ignore all the .catkin_tools stuff from commits

Reply:
you won't have to worry about this for the software repo but just for future reference